oe_theme.theme

```diff
@@ -425,6 +438,22 @@ function oe_theme_preprocess_input__checkbox(array &$variables): void {
   _oe_theme_preprocess_input_label_wrapper($variables);
 }
 
+/**
+ * Implements template_preprocess_input__date().
+ */
+function oe_theme_preprocess_input__date(array &$variables): void {
+  $php_date_format = $variables['#attributes']['data-drupal-date-format'] ?? 'Y-m-d';
```

Comment:
There is no key #attributes, just attributes, and the value of data-drupal-date-format seems to be an array, so we always end up using the default Y-m-d